<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SaveEditor</class>
 <widget class="QDialog" name="SaveEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Save Editor</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="import_raw_save">
       <property name="text">
        <string>Import Raw Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="import_vba_mgba_save">
       <property name="text">
        <string>Import VBA/mGBA Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="import_gameshark_save">
       <property name="text">
        <string>Import GameShark Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="export_raw_save">
       <property name="text">
        <string>Export Raw Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="export_vba_mgba_save">
       <property name="text">
        <string>Export VBA/mGBA Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="export_gameshark_save">
       <property name="text">
        <string>Export GameShark Save</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QComboBox" name="selected_slot_index">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>Save Slot 1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Save Slot 2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Save Slot 3</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1000</width>
        <height>628</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="owned_figurines_layout"/>
       </item>
       <item>
        <layout class="QFormLayout" name="owned_items_layout"/>
       </item>
       <item>
        <layout class="QVBoxLayout" name="flags_layout"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
